### DISCUSSION TITLE
Under high concurrency, the Docker mem used keeps increasing. After the heavy traffic ends, the Docker mem user returns to the normal level

### DISCUSSION THREAD

**Discussion by wind0926** [ben-manes/caffeine#826] [Q&A] [answered]
public AsyncLoadingCache<Integer, Object> test() {
    return Caffeine.newBuilder()
            // 设置最后访问后间隔多久淘汰
            .expireAfterAccess(60 * 60, TimeUnit.SECONDS)
            // 缓存的最大条数
            .maximumSize(30)
            .recordStats()
            .buildAsync(new AsyncCacheLoader<Integer, Object>() {
                @Override
                public CompletableFuture<Object> asyncLoad(Integer key, Executor executor) {
                    return CompletableFuture.supplyAsync(() -> {
                        //TODO
                        return null;
                    }, executor);
                }

                @Override
                public CompletableFuture<Map<Integer, Object>> asyncLoadAll(Iterable<? extends Integer> keys, Executor executor) {
  
                    return CompletableFuture.supplyAsync(() -> {
                        //TODO
                       return null;
                    }, executor);
                }
            });
}

**Comment by ben-manes** [comment, marked as answer]
Please provide a meaningful bug report. Please do not keep creating these incomplete issues. You should only report issues after doing an investigation with data about what problems you are seeing.
